script/gen-service-worker.js

```diff
@@ -9,11 +9,11 @@ Expects home-assistant-polymer repo as submodule of HA repo.
 Creates a caching service worker based on the CURRENT content of HA repo.
 Output service worker to build/service_worker.js
 */
```

Comment:
I'm not too familiar with the caching service worker. Should we be adding translations here?